### PR TITLE
Added validation tests for API

### PR DESCRIPTION
**Pull request by png331**

### CODE REVIEW

File: src/MedicLabAPI/test/models/user_test.rb [hunk 1]
```diff
@@ -1,7 +1,28 @@
 require "test_helper"
 
-class UserTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
-end
+# class UserTest < ActiveSupport::TestCase
```
Comment (marulya) [inline src/MedicLabAPI/test/models/user_test.rb:3]: Generally, no commented-out code should go in the repo. Can you please finish the tests before merging them?

File: src/MedicLabAPI/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+version: "3.7"
```
Comment (marulya) [inline src/MedicLabAPI/docker-compose.yml:1]: We should probably move that file to the `src` folder as it should describe all apps. Here we can define all we need:
1. Database;
2. API;
3. Admin management App;
4. UI.

The above will allow for bringing up the entire environment with a single action.

File: src/MedicLabAPI/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,13 @@
+FROM ruby
```
Comment (marulya) [inline src/MedicLabAPI/Dockerfile:1]: Do we need Docker-related items in this PR? We should probably have only unit test things here. Then we can do another PR for the Docker things.

Comment (png331) [reply src/MedicLabAPI/Dockerfile:1]: Sure. Will make the changes
